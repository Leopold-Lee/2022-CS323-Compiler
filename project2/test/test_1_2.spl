int test_1_2()
{
    int 111_go;
    float id_3;
    return 0
}
