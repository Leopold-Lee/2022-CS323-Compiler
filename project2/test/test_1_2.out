Error type A at Line 3: unknown lexeme 111_go
Error type B at Line 5: Missing semicolon ';'
